src/lando/ui/jobs.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from django.conf import settings
+from django.http import HttpRequest, HttpResponse, JsonResponse
+from django.shortcuts import get_object_or_404, redirect
+from django.template.response import TemplateResponse
+
+from lando.api.legacy.treestatus import (
+    TreeStatusCommunicationException,
+    TreeStatusError,
+)
+from lando.headless_api.models.automation_job import AutomationJob
+from lando.main.models.landing_job import JobStatus, LandingJob
+from lando.main.models.worker import Worker, WorkerType
+from lando.ui.views import LandoView
+from lando.utils import treestatus
+
+logger = logging.getLogger(__name__)  # noqa: F821
+
+
+class JobView(LandoView):
+    pass
+
+
+class LandingQueueView(JobView):
+    def get(self, request: HttpRequest) -> HttpResponse:
+        jobs = LandingJob.job_queue_query().all()
+        data = [
+            {
+                "created_at": j.created_at,
+                "id": j.id,
+                "url": f"{settings.SITE_URL}/landings/{j.id}",
+                "repository": j.target_repo.short_name,
+                "requester": j.requester_email,
+                "revisions": [
+                    f"{settings.PHABRICATOR_URL}/D{r.revision_id}" for r in j.revisions
+                ],
+                "status": j.status,
+                "updated_at": j.updated_at,
+            }
+            for j in jobs
+        ]
```

Comment:
I think this JSON structure would make more sense as an instance method on `LandingJob`, so this would become `data = [job.landing_queue_detail() for job in jobs]`.